src/main/scala/edu/berkeley/ce/rockslicing/InputProcessor.scala

```diff
@@ -24,10 +23,23 @@ object InputProcessor {
     val globalOrigin = globalOriginTokens map(_.get)
     val globalOriginArr = Array(globalOrigin(0), globalOrigin(1), globalOrigin(2))
 
-    val remainingLines = lines.tail
-    val transitionIndex = remainingLines.indexWhere(_._1 == "%")
+    val boundingBoxTokens = boundingBoxLine.split(" ") map {x => Try(x.toDouble) }
```

Comment:
For consistent formatting, you need a space before the `x`, i.e. `map {x ...` should become `map { x ...`.